SU-HSDO/suhumsci
# SHS-5492: Fix bug in hb-stretch-vertical-linked-cards

# READY FOR REVIEW

## Summary
Fix `hb-stretch-vertical-linked-cards` class to cause the cards to be the same height and the titles to all be the same height

## Need Review By (Date)
02/28

## Urgency
high

## Steps to Test
1. Go to https://english.suhumsci.loc/
2. Log in and go to https://english.suhumsci.loc/node/2/layout
3. Remove the `.hb-raised-cards--uniform-height` class in the `Vertical Linked Cards`
4. Confirm the cards and their titles have the same height

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
